test/atat-web-api.test.ts

```diff
@@ -0,0 +1,17 @@
+// import * as cdk from 'aws-cdk-lib';
+// import { Template } from 'aws-cdk-lib/assertions';
+// import * as AtatWebApi from '../lib/atat-web-api-stack';
+
+// example test. To run these tests, uncomment this file along with the
+// example resource in lib/atat-web-api-stack.ts
+test('SQS Queue Created', () => {
```

Comment:
Is this boilerplate or are we trying to actually have proper unit tests to validate our templates? Would be pretty cool if it's useful and not a huge burden.

Reply:
I really want to have real tests! We don't have any infrastructure defined in this PR and this _particular_ code is boilerplate (from a `cdk init`) but the `aws-cdk-lib/assertions` library is pretty awesome!

Reply:
Ya know what, let's get off to to a good start on this reset. I will define some basic infra to kick off at least a few tests